### DISCUSSION TITLE
关于IC2中使用<TNT>和引燃<苦力怕>均能破坏<防爆门>bug

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#32] [【漏洞反馈】Bugs Report]
使用模组版本：1.industrialcraft-2-2.9.84+ex119-1.19.2-forge
2.[1.19.2]Jei物品管理器(forge11.3.0.262)

游戏版本：1.19.2 forge-43.2.0 Op-I1-pre4

在游戏中玩家可以使用TNT和引燃<苦力怕>破坏<防爆门>且<防爆门>不掉落。
<防爆门>材质错误，但具备<门>的所有性质。

**Comment by AbbottMc** [comment]
爆炸后，强化门底部方块如果被爆炸破坏，那么强化门会因为底部方块消失而被破坏，这个是正常现象。
建议在实际使用过程中使用高爆炸抗性的方块作为强化门的基地，保证强化门不会因为门的特性而被轻易破坏
